source/dsl/DangerUtilsDSL.ts

```diff
@@ -5,7 +5,7 @@
 export interface DangerUtilsDSL {
 
   /**
-   * Creates an HTML link.
+   * Creates a HTML link.
```

Comment:
ooooh. controversial.  I guess this depends on how you pronounce 'H'.  I say 'aitch', so 'an' makes sense.  I guess it makes sense if you say 'haitch', or pronounce 'HTML' in some other way.